buku.py

```diff
@@ -2068,12 +2082,27 @@ def prompt_help(file=sys.stdout):
 
 ''')
 
+    @staticmethod
+    def is_colorstr(arg):
+        """Check if a string is a valid color string."""
+        try:
+            assert len(arg) == 4
+            for c in arg:
+                assert c in COLORMAP
+        except AssertionError:
+            raise argparse.ArgumentTypeError('%s is not a valid color string' % arg)
```

Comment:
If an invalid color string is set as the env variable (e.g., `ixejJ`), the raised error message is `buku.py: error: argument --colors: ixejJ is not a valid color string`. Maybe not a problem, but this might be confusing since `--colors` is not used in that case by the user.